code/__HELPERS/global_lists.dm

```diff
@@ -46,6 +46,11 @@
 		GLOB.surgeries_list += new path()
 	sortList(GLOB.surgeries_list, /proc/cmp_typepaths_asc)
 
+	// Hair Gradients - Initialise all /datum/sprite_accessory/hair_gradient into an list indexed by gradient-style name
+	for(var/path in subtypesof(/datum/sprite_accessory/hair_gradient))
+		var/datum/sprite_accessory/hair_gradient/H = new path()
```

Comment:
this is a minor nitpick but initial() abuse might be usable here to cut down on news and object refs floating about